lib/storage/src/content_manager/consensus_manager.rs

```diff
@@ -94,6 +97,8 @@ pub struct ConsensusManager<C: CollectionContainer> {
     consensus_thread_status: RwLock<ConsensusThreadStatus>,
     /// Consensus thread errors, changed by the consensus thread
     message_send_failures: RwLock<HashMap<String, MessageSendErrors>>,
+    /// Last time we attempted to update the peer metadata
+    last_peer_metadata_update: Mutex<Option<Instant>>,
```

Comment:
```suggestion
    next_peer_metadata_update_attempt: Mutex<Instant>,
```

Reply:
Sure. I like that we can remove the option this way.